ocqa/.htaccess

```diff
@@ -25,3 +25,27 @@ RewriteRule ^ocqa.html$ https://sebseis.github.io/OCQA/ [R=302,NE,L]
 
 # Default response: html
 RewriteRule ^$ https://sebseis.github.io/OCQA/ [R=303,NE,L]
+
+# Link to Modules, already described in main document
+# Catalog Module
+RewriteRule ^catalog.ttl$ https://sebseis.github.io/OCQA/Modules/Catalog/OCQA-Catalog.ttl [R=302,NE,L]
+RewriteRule ^catalog.html$ https://sebseis.github.io/OCQA/Modules/Catalog/ [R=302,NE,L]
+# Contract Module
+RewriteRule ^contract.ttl$ https://sebseis.github.io/OCQA/Modules/Contract/OCQA-Contract.ttl [R=302,NE,L]
+RewriteRule ^contract.html$ https://sebseis.github.io/OCQA/Modules/Contract/ [R=302,NE,L]
+# Regulation Module
+RewriteRule ^regulation.ttl$ https://sebseis.github.io/OCQA/Modules/Regulation/OCQA-Regulation.ttl [R=302,NE,L]
+RewriteRule ^regulation.html$ https://sebseis.github.io/OCQA/Modules/Regulation/ [R=302,NE,L]
+# Rule Module
+RewriteRule ^rule.ttl$ https://sebseis.github.io/OCQA/Modules/Rule/OCQA-Rule.ttl [R=302,NE,L]
+RewriteRule ^rule.html$ https://sebseis.github.io/OCQA/Modules/Rule/ [R=302,NE,L]
+
+# Link to Extensions, already described in main document
+# Screed Extension
+RewriteRule ^screed.ttl$ https://sebseis.github.io/OCQA/Extensions/Screed/OCQA-Screed.ttl [R=302,NE,L]
+RewriteRule ^rule.html$ https://sebseis.github.io/OCQA/Extensions/Screed/ [R=302,NE,L]
+# ThermalInsulation Extension
+RewriteRule ^thermalInsulation.ttl$ https://sebseis.github.io/OCQA/Extensions/Insulation/OCQA-ThermalInsulation.ttl [R=302,NE,L]
+RewriteRule ^rule.html$ https://sebseis.github.io/OCQA/Extensions/Insulation/ [R=302,NE,L]
```

Comment:
```suggestion
RewriteRule ^thermalInsulation.html$ https://sebseis.github.io/OCQA/Extensions/Insulation/ [R=302,NE,L]
```

Reply:
Thank you for being so helpful. You were right; it is a typo. I am not familiar with the htaccess files and used language, and I hope that the changes will work as I wanted. In our case, we used some modules and extensions, which we try to provide for users.